src/main/java/teammates/ui/webapi/action/EditCourseAction.java

```diff
@@ -12,7 +12,7 @@
 /**
  * Action: Save edited course details.
  */
-public class SaveCourseEditDetailsAction extends Action {
+public class EditCourseAction extends Action {
```

Comment:
SaveCourseAction.

Just to be consistent as we have `SaveFeedbackSessionAction`, `SaveFeedbackQuestionAction` etc.